src/System Application/App/Azure Storage Services Authorization/src/StorServAuthImpl.Codeunit.al

```diff
@@ -62,6 +62,16 @@ codeunit 9063 "Stor. Serv. Auth. Impl."
         exit(StorServAuthReadySAS);
     end;
 
+    [NonDebuggable]
+    procedure ReadySAS(SASToken: SecretText): Interface "Storage Service Authorization"
```

Comment:
You can directly remove the old method `ReadySAS(SASToken: Text)`.
The facade will automatically call this, as `Text` can be treated as `SecretText`

Reply:
And remove the `[NonDebuggable]` on this method